README.md

```diff
@@ -182,6 +182,27 @@ If you don't have a scanner on your phone, please install:
 
 - [ST BLE Profile](https://play.google.com/store/apps/details?id=com.stm.bluetoothlevalidation) for Android.
 
+
+## Using bare metal profile
+
+Example with NUCLEO_WB55RG:
```

Comment:
Can you add one sentence explaining what the bare metal is (with a reference to it: https://os.mbed.com/docs/mbed-os/v6.10/bare-metal/using-the-bare-metal-profile.html) and what are the tradeoffs ?